sdk/servicebus/Azure.Messaging.ServiceBus/tests/Infrastructure/ServiceBusLiveTestBase.cs

```diff
@@ -8,8 +8,7 @@
 
 namespace Azure.Messaging.ServiceBus.Tests
 {
-    [Category(TestCategory.Live)]
-    [Category(TestCategory.DisallowVisualStudioLiveUnitTesting)]
+    [LiveOnly(true)]
```

Comment:
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core.TestFramework/src/LiveTestBase.cs#L14

Reply:
See the param that I am passing..